aws-datastore/src/main/java/com/amplifyframework/datastore/syncengine/Merger.kt

```diff
@@ -66,8 +66,14 @@ internal class Merger(
                 return@rxCompletable
             }
 
+            // Keep the models with the highest version for each ids, abandon the rest
+            var modelsWithUniqueId = modelsWithMetadata.groupBy { modelWithMetadata -> modelWithMetadata.model.primaryKeyString }
+                .map {group ->
+                    group.value.maxBy { dupModels -> dupModels.syncMetadata.version ?: 0 }
+                }.toList()
```

Comment:
don't need `toList()`. It is already a list

Reply:
removed `toList()`